### DISCUSSION TITLE
Pokémon Brilliant Diamond and Shining Pearl

### DISCUSSION THREAD

**Discussion by APKROB** [skyline-emu/skyline#610] [Game Requests]
This game has big potential to work perfectly on Skyline i real believe this dev of skyline is Great and hard working. ☺️ I can't wait to see this kind of game will be running on mobile.. more power,!

**Comment by Nemomando** [comment]
Também acho que  seria muito bom   no skyline   com certeza é um título que poderia ser aproveitado por muita gente!
